falcon_multiqc/commands/delete_tables.py

```diff
@@ -0,0 +1,13 @@
+import click
+from database.crud import recreate_database
+
+# Command for removing all data entries in database, tables will still remain.
+
+
+@click.command()
+def cli():
+    """Deletes all database entries"""
+    
+    print("Are you sure you want to delete all data entries in the database?")
```

Comment:
Tiny comment, can you use click.echo for printing and for input maybe https://click.palletsprojects.com/en/7.x/prompts/ ?

It should do the same thing, but click will handle it it better.

Reply:
yup fixed!